packages/destination-actions/src/destinations/fullstory/request-params.ts

```diff
@@ -35,6 +35,8 @@ const defaultRequestParams = (settings: Settings, relativeUrl: string): RequestP
   }
 }
 
+// TODO(nate): Remove "region specific" comments
```

Comment:
I'm planning to open a fast follow PR that cleans up artifacts like this from earlier iterations. Omitted here to keep PR surface smaller and more focused on the new `trackEvent` action.